### DISCUSSION TITLE
Improve hide players in playtimetop

### DISCUSSION THREAD

**Discussion by NoltoxGit** [Zrips/CMI#6856] [Enhancement]
Hello !

Currently you can hide players in the _config.yml_ of the plugin.
But I think it's a bit "lame", because when a player changes his nickname, he's back in the ranking

What I propose is to add a permission, like: `cmi.command.playtime.hide` which would allow (for example if we have LuckPerms) to assign this directly to a group without having to change the nicknames in the _config.yml_ every time.

Ty,
~ @NoltoxGit ~

**Comment by CloudeLecaw** [comment]
If your server has onlinemode-true u can add also his uuid instead of his name.
Than his nane doesnt matter

**Comment by mrfloris** [comment]
These are now an option in config.yml
```yaml
  # Time in days for player to be offline when he will be excluded from playtimetop list
  # Set to 0 if you want to disable this
  PlaytimeTopOffline: 0
```
This would at least help with those 'dead' fake accounts that aren't active, and you don't have to add their names to the exclude list. 

You can already add the name to the list, it would be nice if it extends to also take UUID, I don't think a permission is needed really.
